unit-tests/src/test/scala/scala/scalanative/posix/TimeTest.scala

```diff
@@ -2,6 +2,7 @@ package scala.scalanative.posix
 
 import org.junit.Test
 import org.junit.Assert._
+import org.junit.Assume.assumeFalse
```

Comment:
See suggestion below about flipping sense of isDst.  That would
flip this include.